src/Reviews.jsx

```diff
@@ -0,0 +1,39 @@
+import { useState, useEffect } from 'react'
+import { getReviews } from '../utils'
+import { Link } from 'react-router-dom'
+
+function Reviews () {
+    const [reviewList, setReviewList] = useState([])
+    
+    
+    useEffect(() => {
+        getReviews().then((data) => {
+          setReviewList(data)
+        })
+      })
+    
+    return (
+        <main>
+            <h1>Welcome to Reviews</h1>
+            <ul className='navList'>
+        {reviewList.map((item) => {   
+            let num = item.review_id
+            const link = `/reviews/${num}`
```

Comment:
You can write the route path directly in the link tag. There is no need to add a variable as this is bloating your code.